protos/Robot_Echo/SRRobot.proto

```diff
@@ -14,980 +14,979 @@ PROTO SRRobot [
     translation IS translation
     rotation IS rotation
     controllerArgs IS controllerArgs
-      children [
-    DEF bump_sensors Group {
-      children [
-        TouchSensor {
-          translation 0 0.09 -0.03
-          rotation 0 1 0 -3.1415926535897705
-          children [
-            Shape {
-              appearance PBRAppearance {
-                baseColor 0.5 0 0
-                roughness 0.6980390000000001
-              }
-              geometry DEF BUMPER Box {
-                size 0.14 0.01 0.01
-              }
-            }
-          ]
-          name "front bump sensor"
-          boundingObject Transform {
-            translation 0 0 0
+    children [
+      DEF bump_sensors Group {
+        children [
+          TouchSensor {
+            translation 0 0.09 -0.03
+            rotation 0 1 0 -3.14
```

Comment:
not sure this float reduction is desirable? It looks a lot like the value here is π, which we probably don't want to truncate.

Reply:
The arena is only rotated north to 4 d.p. precision so it seems unnecessary to have this rotation to 16 d.p., ultimately this is a change of under 0.1 degrees.